### PR TITLE
FBFW-82 & 119 QA Feedback Fixes

### PR DESCRIPTION
**Pull request by GCroom**
## Test Link
https://623c9cab8335db00caf34566--flamboyant-franklin-5232af.netlify.app/

- Moved the Out of Bounds territories to a separate "g" id so they would load last (the board was not centered because they were loading randomly within the land and sea territories)

| before | after |
| --- | --- |
| image | image |

- Updated the Piedmont label and unit placement
   - Figma: https://www.figma.com/file/JnpO52R22kG3lB2wexs9rB/webDiplomacy?node-id=3842%3A34366

| before | after |
|---|---|
| image | image |

### CODE REVIEW

**Review by nebevets — COMMENTED** [review]
LGTM

**Review by nebevets — APPROVED** [review]

**Comment by bryancodazen** [comment]
@GCroom can you merge this PR in? https://github.com/codazen/webDiplomacy/pull/56

File: beta-src/src/components/map/variants/classic/components/WDBoardMap.tsx [hunk 1]
```diff
@@ -362,25 +362,37 @@ const WDBoardMap: React.FC = function (): React.ReactElement {
...
         <WDTerritory territoryMapData={VIENNA} />
         <WDTerritory territoryMapData={WALES} />
         <WDTerritory territoryMapData={WARSAW} />
         <WDTerritory territoryMapData={YORK} />
         {/* These need to be last so the labels and units will appear */}
         <WDTerritory territoryMapData={NAPLES} />
+        <WDTerritory territoryMapData={PIEDMONT} />
         <WDTerritory territoryMapData={ROME} />
         <WDTerritory territoryMapData={TUSCANY} />
         <WDTerritory territoryMapData={PIEDMONT} />
```
Comment (bryancodazen) [inline beta-src/src/components/map/variants/classic/components/WDBoardMap.tsx:375]: Seems like you have piedmont twice here.

**Review by bryancodazen — CHANGES_REQUESTED** [review]
Small change

**Review by bryancodazen — APPROVED** [review]
LOVELY

**Pull request merged**